test/common/stats/symbol_table_impl_test.cc

```diff
@@ -507,6 +507,20 @@ TEST_P(StatNameTest, MutexContentionOnExistingSymbols) {
   }
 }
 
+TEST_P(StatNameTest, SharedStatNameStorageSet) {
```

Comment:
nit: Rename this to describe the aspect of the sharedstatenamestorageset we're testing.

Reply:
done